### PR TITLE
Added filepath to shipment response

### PR DESCRIPTION
**Pull request by joshplumley**

### CODE REVIEW

File: src/packingSlip/model.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ const schema = new Schema({
       destinationCode: String,
 
       // Upload URL for router documents
-      routerUploadFilePath: String,
+      confirmShipmentFilePath: String,
```
Comment (jbolles94) [inline src/packingSlip/model.js:40]: This should not have changed.

File: src/shipment/controller.js [hunk 2]
```diff
@@ -316,7 +316,7 @@ async function createShipment(
 
     await Promise.all(
       packingSlip.items.map((item) => {
-        itemNoRouter |= item.routerUploadFilePath === undefined;
+        itemNoRouter |= item.confirmShipmentFilePath === undefined;
```
Comment (jbolles94) [inline src/shipment/controller.js:319]: This should not have changed.

File: src/shipment/controller.js [hunk 3]
```diff
@@ -717,7 +717,7 @@ async function editOne(req, res) {
         isDueBack, //for incomingDeliveries
         isDueBackOn, //for incomingDeliveries
         shipmentImages,
-        routerUploadFilePath,
+        confirmShipmentFilePath,
```
Comment (jbolles94) [inline src/shipment/controller.js:720]: If we are changing this, then we need a corresponding FE change.

**Review by jbolles94 — CHANGES_REQUESTED** [review]

**Comment by joshplumley** [comment]
@jbolles94 fixed the names.

**Review by jbolles94 — APPROVED** [review]

**Review by jbolles94 — APPROVED** [review]

**Pull request merged**